activity.py

```diff
@@ -412,39 +412,39 @@ def _graph_from_reader(self, reader):
         horizontal, vertical = reader.get_labels_name()
 
         # Load the data
-        for row  in chart_data:
-            self._add_value(None,
-                            label=row[0], value=float(row[1]))
+        for row in chart_data:
+            self._add_value(None, label=row[0], value=float(row[1]))
 
             self.update_chart()
 
     def _add_value(self, widget, label="", value="0.0"):
         data = (label, float(value))
-        if not data in self.chart_data:
+        if data not in self.chart_data:
             pos = self.labels_and_values.add_value(label, value)
             self.chart_data.insert(pos, data)
             self._update_chart_data()
 
-
     def _remove_value(self, widget):
         value = self.labels_and_values.remove_selected_value()
         self.chart_data.remove(value)
         self._update_chart_data()
 
     def __import_freespace_cb(self, widget):
-            reader = FreeSpaceReader()
-            self._graph_from_reader(reader)
+        reader = FreeSpaceReader()
+        self._graph_from_reader(reader)
 
     def __import_journal_cb(self, widget):
         reader = JournalReader()
         self._graph_from_reader(reader)
 
     def __import_turtle_cb(self, widget):
-        matches_mime_type, file_path, title = self._object_chooser(
-            'application/x-turtle-art', _('Turtle'))
-        if matches_mime_type:
-            reader = TurtleReader(file_path)
-            self._graph_from_reader(reader)
+        try:
+            matches_mime_type, file_path, title = self._object_chooser('application/x-turtle-art', _('Turtle'))
```

Comment:
Maybe you can split the line into two, exceeds character limit (79), on the line. (E501, Flake) . Atm it has 111

Reply:
Done in 8181469